README.md

```diff
@@ -52,8 +52,9 @@ To build SONiC installer image and docker images, run the following commands:
 
  **NOTE**:
 
-- We recommend reserving 50G free space to build one platform.
+- Recommend reserving 50G free space to build one platform.
 - Use ```http_proxy=[your_proxy] https_proxy=[your_proxy] make``` to enable http(s) proxy in the build process.
+- Add your user account to ```docker``` group and use your user account to make. ```root``` or ```sudo``` are not supported.
```

Comment:
Single backticks are all that is required to inline single-line code. Triple backticks are only required for multi-line code blocks.